app/utils/avoid_native_bridge.js

```diff
@@ -0,0 +1,10 @@
+
+import {Platform} from 'react-native';
+
+export default async function avoidNativeBridge(runOptimized, optimized, fallback) {
```

Comment:
does this mean that iOS won't have stored anything like Android does? will that not cause some difficulties reading the code?

Reply:
makes perfect sense now that I'm reading it again

Reply:
I'll add documentation